homeassistant/components/zimi/quality_scale.yml

```diff
@@ -0,0 +1,26 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: >
+      There are no service actions.
+  appropriate-polling:
+    status: done
+    comment: >
+      There is no polling of the entities.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
```

Comment:
please add the standard include for integration removal as well that describes the basics how to delete an integration (check e.g. tedee)

Reply:
Added the standard text - there is no need to remove the integration from the zimi app so that sentence was not included.   https://github.com/home-assistant/home-assistant.io/pull/35578/commits/4ea6a1eb2d5a49a5ae0b5b0bb49ce86efd7e9e20